QgisModelBaker/ui/edit_command.ui

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>EditCommandDialog</class>
+ <widget class="QDialog" name="EditCommandDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>413</width>
+    <height>266</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Edit command</string>
+  </property>
+  <layout class="QGridLayout" name="gridLayout">
+   <item row="3" column="0" colspan="2">
+    <widget class="QDialogButtonBox" name="button_box">
+     <property name="orientation">
+      <enum>Qt::Horizontal</enum>
+     </property>
+     <property name="standardButtons">
+      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
+     </property>
+    </widget>
+   </item>
+   <item row="2" column="0" colspan="2">
+    <widget class="QPlainTextEdit" name="command_edit"/>
+   </item>
+   <item row="0" column="0" colspan="2">
+    <widget class="QLabel" name="label_warning">
+     <property name="text">
+      <string>There will be dragons...</string>
```

Comment:
```suggestion
      <string>Edit the command below to customize the command for ili2db. More information can be found <a href="https://github.com/claeis/ili2db/blob/master/docs/ili2db.rst">in the ili2db documentation</a>.<br>Please note that QGIS Model Baker is optimized for the default ili2db parameters, if these parameters are modified, QGIS Model Baker might encounter problems and not provide the best results.</string>  
```

I guess there needs to be some escaping (just copy to qtdesigner and let it do the work might be easiest) and probably some handling for link opening in a browser.